jupyter_server/serverapp.py

```diff
@@ -41,17 +41,16 @@
 from jupyter_server.transutils import trans, _
 from jupyter_server.utils import secure_write, run_sync
 
-# check for tornado 3.1.0
+# the minimum viable tornado version: needs to be kept in sync with setup.py
+MIN_TORNADO = (6, 1, 0)
+
 try:
     import tornado
-except ImportError as e:
-    raise ImportError(_("The Jupyter Server requires tornado >= 4.0")) from e
-try:
-    version_info = tornado.version_info
-except AttributeError as e:
-    raise ImportError(_("The Jupyter Server requires tornado >= 4.0, but you have < 1.1.0")) from e
-if version_info < (4,0):
-    raise ImportError(_("The Jupyter Server requires tornado >= 4.0, but you have %s") % tornado.version)
+    assert tornado.version_info >= MIN_TORNADO
+except (ImportError, AttributeError, AssertionError) as e:  # pragma: no cover
+    raise ImportError(
+        _("The Jupyter Server requires tornado >=%s.%s.%s", *MIN_TORNADO)
```

Comment:
This still isn't handling things correctly.  How about we just cut to the chase?
```suggestion
        _(f"The Jupyter Server requires tornado >= {MIN_TORNADO[0]}.{MIN_TORNADO[1]}.{MIN_TORNADO[2]}")
```